templates/partials/content-section.html

```diff
@@ -0,0 +1,55 @@
+<div class="container-fluid content-section">
+    <div class="row mt-5">
+        {% if template %}
```

Comment:
All of these if/elif blocks should be their own partials

Reply:
Moved all these to the index to be used as partials.  
Deleted content-section.html.